### DISCUSSION TITLE
How to create a pre-filled key/value?

### DISCUSSION THREAD

**Discussion by Zyles** [directus/directus#2752] [Archive]
I need to create a key value type for the days of the week, and the user will fill in the opening hours.

What I would like is the days of the week pre-filled and the user only has to fill in the opening hours.

Basically like a table.

monday | ...
tuesday | ...
wednesday | ...
...

Can this be done?

**Comment by rijkvanzanten** [comment]
Are you imagining this to save as a JSON blob or separate columns? I can imagine you might want the data split across columns for search/filter reasons?

As of right now, there's no such "fill in the blanks" interface, but could definitely be an interesting one to look at adding 🙂

**Reply by Zyles** [reply]
The underlying data should just be a JSON key value store.

I solved it for now with a JSON type and a template.

But it would be better for the user to have a two column interface where the first key value is prefilled (and unable to be edited) and the second is blank.
